source/plugins/go/src/oms.go

```diff
@@ -502,6 +520,46 @@ func populateExcludedStderrNamespaces() {
 	}
 }
 
+func populateIncludedStdoutSystemResource() {
```

Comment:
we should refactor the code in populateIncludedStdoutSystemResource & populateIncludedStderrSystemResource to single common function since the logic same in both the functions.